src/ui/czi-editor.css

```diff
@@ -123,7 +123,7 @@
   box-shadow: unset;
   min-height: unset;
   padding: unset;
-  width: unset;
+  width: 78vw;
```

Comment:
Shouldn't this be `100%`? It should fill the parent container. This looks like it would break the comments ui.